source/containers.tex

```diff
@@ -2639,17 +2639,18 @@
   template <class T, size_t N>
   struct array {
     //  types:
-    typedef T&                                    reference;
-    typedef const T&                              const_reference;
-    typedef @\impdefx{type of \tcode{array::iterator}}\itcorr[-1]@                iterator;
-    typedef @\impdefx{type of \tcode{array::const_iterator}}\itcorr[-1]@                const_iterator;
-    typedef size_t                                size_type;
-    typedef ptrdiff_t                             difference_type;
-    typedef T                                     value_type;
-    typedef T*                                    pointer;
-    typedef const T*                              const_pointer;
-    typedef std::reverse_iterator<iterator>       reverse_iterator;
-    typedef std::reverse_iterator<const_iterator> const_reverse_iterator;
+    using value_type             = T;
+    using pointer                = T*;
+    using const_pointer          = const T*;
+    using reference              = T&;
+    using const_reference        = const T&;
+    using size_type              = size_t;
+    using difference_type        = ptrdiff_t;
+    using iterator               = @\impdefx{type of \tcode{array::iterator}}\itcorr[-1]@;
```

Comment:
I'll admit, I am mostly moving text around and don't recognize all of the escape codes ;)
Will update, thanks.